package.json

```diff
@@ -39,18 +39,23 @@
     "axios": "^0.24.0",
     "bignumber.js": "^9.0.2",
     "bip39": "^3.0.4",
+    "browserify-zlib": "^0.2.0",
```

Comment:
why are we bringing in so many polyfills? I suspect that all officialls supported browsers have support for most of these things out of the box.

Reply:
The polyfills are part of #605, not this PR specifically. They're required because Webpack 5 stopped auto-polyfilling Node.js stuff.